net/server.go

```diff
@@ -499,6 +499,15 @@ func peerIDFromContext(ctx context.Context) (libpeer.ID, error) {
 	return pid, nil
 }
 
+const (
+	dockeyPrefix     = "d/"
+	collectionPrefix = "c/"
+)
+
+func newTopic(prefix, topic string) string {
```

Comment:
thought: This func is a bit strange to me, it is basically an alias of the `+` operator, and I dont see much benefit in it.

Might be nicer to replace it with two functions, `newDockeyTopic` and `newCollectionTopic`.

Reply:
I rearranged a little bit to improve the rollback. Let me know what you think.

Reply:
I don't know if it's go's convention or something, but it seems like usually function with prefix `new` should allocate/create a new object. Maybe `makeTopic`, `toTopic` or something similar would fit better.
On the other hand I support delegating the logical blocks to separate function (even if they just concatenate strings) because if we change the way topic are composed we will change it only in one place